docs/cudf/source/conf.py

```diff
@@ -308,7 +308,10 @@ def clean_all_xml_files(path):
     "dlpack": ("https://dmlc.github.io/dlpack/latest/", None),
     "nanoarrow": ("https://arrow.apache.org/nanoarrow/latest", None),
     "numpy": ("https://numpy.org/doc/stable", None),
-    "pandas": ("https://pandas.pydata.org/docs/", None),
+    "pandas": (
+        "https://pandas.pydata.org/pandas-docs/stable/",
+        "https://pandas.pydata.org/pandas-docs/stable/objects.inv",
```

Comment:
I think that this is superfluous. The key fix is adding the `stable/` I assume since previously we were looking in the wrong place. `intersphinx` will find the inventory at this location automatically AFAIK.
```suggestion
        None,
```

Reply:
![Screenshot 2024-04-17 at 9 46 00 PM](https://github.com/rapidsai/cudf/assets/17162724/5afeaa4c-de13-4f84-9be1-53a02a2bc460)

Yep still works as expected

Reply:
Thanks!